### PR TITLE
Release 2023 0 fix links

### PR DESCRIPTION
**Pull request by michalkulakowski**

### CODE REVIEW

File: client/cpp/kserve-api/README.md [hunk 1]
```diff
@@ -438,7 +438,7 @@ Requests per second: 457.283
 Download the model files and store them in the `models` directory
 ```Bash
 mkdir -p models/resnet/1
-curl https://storage.openvinotoolkit.org/repositories/open_model_zoo/2022.1/models_bin/2/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.bin https://storage.openvinotoolkit.org/repositories/open_model_zoo/2022.1/models_bin/2/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.xml -o models/resnet/1/resnet50-binary-0001.bin -o models/resnet/1/resnet50-binary-0001.xml
+curl https://storage.openvinotoolkit.org/repositories/open_model_zoo/2023.0/models_bin/2/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.bin https://storage.openvinotoolkit.org/repositories/open_model_zoo/2022.1/models_bin/2/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.xml -o models/resnet/1/resnet50-binary-0001.bin -o models/resnet/1/resnet50-binary-0001.xml
```
Comment (dkalinowski) [inline client/cpp/kserve-api/README.md:441]: We should not use bin and xml from different versions

File: demos/multi_faces_analysis_pipeline/python/README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+This document demonstrates how to create complex pipelines using object detection and object recognition models from OpenVINO Model Zoo. As an example, we will use [face-detection-retail-0004](https://github.com/openvinotoolkit/open_model_zoo/blob/2022.1.0/models/intel/face-detection-retail-0004/README.md) to detect multiple faces on the image. Then, for each detected face we will crop it using [model_zoo_intel_object_detection](https://github.com/openvinotoolkit/model_server/tree/releases/2023/0/src/custom_nodes/model_zoo_intel_object_detection) example custom node. Finally, each image face image will be forwarded to [age-gender-recognition-retail-0013](https://github.com/openvinotoolkit/open_model_zoo/blob/2022.1.0/models/intel/age-gender-recognition-retail-0013/README.md) and [emotion-recognition-retail-0003](https://github.com/openvinotoolkit/open_model_zoo/blob/2022.1.0/models/intel/emotions-recognition-retail-0003/README.md) models.
```
Comment (dkalinowski) [inline demos/multi_faces_analysis_pipeline/python/README.md:4]: There are multiple 2022.1.0 links here, should those be adjusted as well?

File: demos/vehicle_analysis_pipeline/python/README.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
...
+This document demonstrates how to create complex pipelines using object detection and object recognition models from OpenVINO Model Zoo. As an example, we will use [vehicle-detection-0202](https://github.com/openvinotoolkit/open_model_zoo/blob/2022.1.0/models/intel/vehicle-detection-0202/README.md) to detect multiple vehicles on the image. Then, for each detected vehicle we will crop it using [model_zoo_intel_object_detection](https://github.com/openvinotoolkit/model_server/tree/releases/2023/0/src/custom_nodes/model_zoo_intel_object_detection) example custom node. Finally, each vehicle image will be forwarded to [vehicle-attributes-recognition-barrier-0042](https://github.com/openvinotoolkit/open_model_zoo/blob/2022.1.0/models/intel/vehicle-attributes-recognition-barrier-0042/README.md) model.
```
Comment (dkalinowski) [inline demos/vehicle_analysis_pipeline/python/README.md:2]: There are 2022.1.0 links in this line, should those be changed as well?

**Review by dkalinowski — APPROVED** [review]

**Review by rasapala — APPROVED** [review]
LGTM

**Pull request merged**